docs/user_guides/tls_proxy.md

```diff
@@ -443,4 +485,22 @@ common:
 
 ## How does the client use tls to connect to milvus
 
-Please refer to [example_tls.py](https://github.com/milvus-io/pymilvus/blob/master/examples/example_tls.py).
\ No newline at end of file
+Client need client.pem, client.key, ca.pem, and these file paths should be noticed when you use the milvus sdk.
```

Comment:
You should note the file paths of  `client.pem`, `client.key`, and `ca.pem` files for the client when using the Milvus SDK.